src/components/CreateNewProjectWizard/CreateNewProjectWizard.js

```diff
@@ -73,9 +73,13 @@ class CreateNewProjectWizard extends PureComponent<Props, State> {
   };
 
   verifyProjectNameUniqueness = (name: string) => {
+    const { projects } = this.props;
+
     // Check to see if this name is already taken
-    const id = getProjectId(name);
-    const isAlreadyTaken = !!this.props.projects[id];
+    const slug = getProjectNameSlug(name);
+    const isAlreadyTaken = !!Object.keys(this.props.projects).some(
+      id => projects[id].name === slug
+    );
```

Comment:
This was something I missed in the last review; had to update it so that you couldn't create a new project with the same path as an existing one.

Reply:
Yes, good catch. I also missed it.
I'm wondering if we also need this check during project import or is it handled differently there? So that you can not import two projects with the same name or should this be possible? If it's possible we need a way to see the path or something inside ProjectPage.
If we can use it for import too we could move this to a service or util file.

Reply:
I think it's ok for import; the project being imported is likely in a different directory, so it doesn't matter (it's OK if two projects share the same name as long as they don't both live in the same parent directory).

One thing I'm not sure about is what happens when you try to create a new project with the same name as a project you removed from Guppy but kept on disk... but I'm not sure why anyone would try and do that.